cylc/flow/remote.py

```diff
@@ -107,6 +107,10 @@ def run_cmd(
         * Else True if the remote command is executed successfully, or
           if unsuccessful and capture_status=True the remote command exit code.
         * Otherwise exit with an error message.
+
+    Exits:
+        1: In the event of certain command errors.
```

Comment:
I think `Exits with code 1 in the event of certain command errors.` might be a little clearer

Reply:
And/or "Exit codes" rather than "Exits".

Reply:
All yours!

Reply:
```suggestion
    Exits with code 1 in the event of certain command errors.
```